### DISCUSSION TITLE
Find sprint weekends

### DISCUSSION THREAD

**Discussion by Casper-Guo** [theOehrly/Fast-F1#540] [Q&A] [answered]
I want to get the round number of all sprint weekends in a season. One solution is to get all the events and then test how many sessions they have. I am wondering if there is a way to do what I want while making less API requests to FastF1.

**Comment by theOehrly** [comment, marked as answer]
The event schedule has an 'EventFormat' column. Therefore, you can get the total count like this

```
import fastf1
schedule = fastf1.get_event_schedule(2024)
(schedule['EventFormat'] == 'sprint_shootout').sum()
```

Is that what you are looking for?

**Reply by Casper-Guo** [reply]
This is great, thanks!
